landoui/static/src/css/components/Footer.scss

```diff
@@ -0,0 +1,3 @@
+.Footer {
```

Comment:
Did you see the `static/src/css/README.md` ? They start with uppercase letters because of the SUIT CSS pattern discussed there (meant to allow us to have components for all our stuff, sorta like we did in React).